spec/models/product_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe Product, type: :model do
+  describe '\'Products\' has db columns' do
+    it {is_expected.to have_db_column :id}
+    it {is_expected.to have_db_column :name}
+    it {is_expected.to have_db_column :price}
+  end
+
+  describe 'Validations' do
+    it {is_expected.to validate_presence_of :name}
+    it {is_expected.to validate_presence_of :price}
+  end
+
+  describe 'Factory' do
+    it 'should have valid Factory' do
```

Comment:
Keep the consistency with the declaration messages, like we touched upon yesterday in the demo, use `is expected to` instead of should